.github/workflows/run_tests.yml

```diff
@@ -4,7 +4,7 @@
 name: Run Staging Service Tests
 
 on:
-  [push, pull_request]
```

Comment:
Maybe I should have changed this to "pull_request" instead, I got tired of seeing double builds.

Reply:
Is PR for testing changes before they're merged, and push was for testing changes in the merged state? Or is push redundant since we shouldn't be pushing directly to develop/master etc?

Reply:
I think we shouldn't be pushing directly

Reply:
Agree there